salsa_dancing_molecules/simulations/simulation.py

```diff
@@ -1,4 +1,68 @@
 """Module for running a simulation."""
+from ..lennardjonesparse import parse_lj_params
+from ase.md.velocitydistribution import MaxwellBoltzmannDistribution
+from ase.md.verlet import VelocityVerlet
+from ase.md.andersen import Andersen
+from ase import units
+from asap3 import Trajectory
+from ..variables import Variables
+
+
+def choose_potential(potential, kim_model, use_asap, atoms):
+    """Change the atoms object according to what potential is used.
```

Comment:
Could we add some more text explaining what this function does? Just changing the potential of the atoms object is not very descriptive in my opinion.

Reply:
We have added an expanded comment to the function.